conda/models/match_spec.py

```diff
@@ -166,7 +167,9 @@ class MatchSpec(object):
         'version',
         'build',
         'build_number',
-        'track_features',
```

Comment:
Oh, I should probably take that out.  The reason 'features' is still there is because, from the email thread today

> > I think an example of using the MatchSpec language to specify a feature for a single package would be a good addition to the "key-value features" section.
> 
> I'll add that.  The reason I hadn't quite yet is because it's the 'provides_features' key that needs to be used, and I'm not 100% comfortable with that.  What I mean is something like `conda install numpy[provides_features='blas=nomkl']`.  When users are giving specs on the command line, it seems more intuitive that they'd want to give the `requires_features` key, since that's what they want their environment to require.  That's opposite from the perspective the solver logic takes though, which is preferring determination of the features that the environment provides.
> 
> In the new matchspec query language, I've wrestled with the idea of just translating the 'features' key to 'provides_features'.  So the command line syntax would then just simply be `conda install numpy[features='blas=nomkl']`.  However, that also might be too much magic.  It also conflates the old 'features' key, which does still exist for legacy reasons.  Happy to hear thoughts here.